app/components/PhotographerSection.js

```diff
@@ -0,0 +1,234 @@
+'use client';
+
+import { useRef, useLayoutEffect } from 'react';
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Marquee from '@/app/components/Marquee';
+import PhotographerCard from '@/app/components/PhotographerCard';
+import EastIcon from '@mui/icons-material/East';
+import { photographers } from '@/app/lib/placeholder-data';
+import { gsap } from 'gsap';
+import { ScrollTrigger } from 'gsap/ScrollTrigger';
+import { Noto_Sans_TC } from 'next/font/google';
+import photographerBg from '../../public/phorographer-bg.png';
+import { Swiper, SwiperSlide } from 'swiper/react';
+import { FreeMode } from 'swiper/modules';
+import 'swiper/css';
+import 'swiper/css/free-mode';
```

Comment:
import 順序可以稍微調整一下，相關性的可以放一起，順序原則上大概是：react 相關 > 第三方套件 > 元件 > Icon + 字型 > 圖片匯入

```suggestion
import 'swiper/css';
import 'swiper/css/free-mode';

import { useRef, useLayoutEffect } from 'react';
import { Swiper, SwiperSlide } from 'swiper/react';
import { FreeMode } from 'swiper/modules';
import Container from '@mui/material/Container';
import Typography from '@mui/material/Typography';
import Box from '@mui/material/Box';
import Link from '@mui/material/Link';

import { gsap } from 'gsap';
import { ScrollTrigger } from 'gsap/ScrollTrigger';

import Marquee from '@/app/components/Marquee';
import PhotographerCard from '@/app/components/PhotographerCard';
import { photographers } from '@/app/lib/placeholder-data';

import { Noto_Sans_TC } from 'next/font/google';
import EastIcon from '@mui/icons-material/East';
import photographerBg from '../../public/phorographer-bg.png';

```